base/challenge-skeleton/README.md

```diff
@@ -1,52 +1,59 @@
 * Check out `make <tab>` for more commands.
 
-## healthcheck
+## Healthcheck
+
+Important details regarding healthchecks:
 
-* The healthcheck needs to run a webserver on port 45281 that responds to /healthz requests.
- * 200 if the challenge is healthy, otherwise an error (e.g. 400).
+* Run the healthcheck on a webserver on port 45281 that responds to `/healthz` requests.
+ * The healthcheck returns 200 if the challenge is healthy, otherwise an error (e.g. 400).
  * You can find an example webserver in `healthcheck/image/healthz.py`.
 * The base image comes preloaded with pwntools.
-* If your exploit is written in python, just drop it in `healthcheck/image/doit.py`.
+* If your exploit is written in Python, drop it in `healthcheck/image/doit.py`.
+
+## Directory layout
 
-# Directory layout:
+The following directories are available:
 
-## /config
+### /config
 
-The config directory holds a few configuration files:
+The `config` directory holds a few configuration files:
 
-* chal.conf: Configure if the challenge should be deployed to the cluster, if it's publicly accessible and if it has a healthcheck.
-* pow.conf: Contains the difficulty of the proof-of-work, 0 means disabled.
-* advanced: The kubernetes config used to deploy the challenge. E.g. you can add a tmpfs here or change the port.
+* `chal.conf`: Configure this file if you are deploying the challenge to the cluster, if it's publicly accessible, and if it has a healthcheck.
+* `pow.conf`: Contains the difficulty of the proof-of-work, 0 means disabled.
+* `advanced`: The Kubernetes config used to deploy the challenge. E.g. you can add a tmpfs here or change the port.
 
-## /challenge
+### /challenge
 
-The challenge binary and anything else challenge related goes in here.
+The `challenge` directory contains the challenge binary and anything else related to the challenge:
 
-* image/: Dockerfile and files of the challenge image.
-* Makefile: You can edit this file if you need to run code before building the docker image, e.g. for building the challenge from source.
+* `image/`: Dockerfile and files of the challenge image.
+* `Makefile`: Edit this file if you need to run code before building the Docker image, e.g. for building the challenge from source.
 
-## /healthcheck
+### /healthcheck
 
-The healthcheck directory is optional. If you don't want to write a healthcheck, feel free to delete it. You really should have healthchecks though :).
+The `healthcheck` directory is optional. If you don't want to write a healthcheck, feel free to delete the directory. However, we strongly recommend that you implement a healthcheck :).
 
-* image/: Dockerfile and files of the healthcheck.
-* Makefile: same as for the challenge directory.
+* `image/`: Dockerfile and files of the healthcheck.
+* `Makefile`: Edit this file if you need to run code before running your healthcheck.
 
-# API contract
+## API contract
```

Comment:
I forgot one point: In your cmdline, you have to call `kctf_setup` as the first command. Would you mind adding that point as part of this pull request?

Reply:
thank you! @goedi02